src/ts/.gitignore

```diff
@@ -0,0 +1,35 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# node
+package-lock.json
```

Comment:
Why removing package lock? Wouldn't be a good idea that everyone has the same exact version?

Reply:
I left the **yarn.lock** versioned for that purpose, I supposed it's enough with that one